spark-cassandra-connector/src/main/scala/com/datastax/spark/connector/cql/CassandraConnectorConf.scala

```diff
@@ -20,7 +20,10 @@ case class CassandraConnectorConf(
   queryRetryCount: Int = CassandraConnectorConf.DefaultQueryRetryCount,
   connectTimeoutMillis: Int = CassandraConnectorConf.DefaultConnectTimeoutMillis,
   readTimeoutMillis: Int = CassandraConnectorConf.DefaultReadTimeoutMillis,
-  connectionFactory: CassandraConnectionFactory = DefaultConnectionFactory
+  connectionFactory: CassandraConnectionFactory = DefaultConnectionFactory,
+  sslEnabled: Boolean = CassandraConnectorConf.DefaultSSLEnabled,
```

Comment:
Could you create some case class to enclose all SSL related settings? then there would be a single field like SSL configuration which would be optional.